main.go

```diff
@@ -1,53 +1,14 @@
 package main
 
 import (
-	"context"
 	"os"
-	"time"
-
-	"github.com/suborbital/subo/subo/release"
-	"github.com/suborbital/subo/subo/util"
 )
 
-const checkVersionTimeout = 500 * time.Millisecond
-
 func main() {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-	versionChan := checkVersion(ctx)
-
 	rootCmd := rootCommand()
 	if err := rootCmd.Execute(); err != nil {
 		os.Exit(-1)
```

Comment:
while here, can you also fix this? The only valid exit codes are 0-255. If this is an error, make it a 1, rather than a negative 1 :)

Reply:
Depending on what the error was with the rootCmd, we could choose between any of the special ones from here: http://www.bic.mni.mcgill.ca/~dale/helppages/BashGuide/advshell/exitcodes.html, or use the catchall 1.